doc/nrf/releases/release-notes-latest.rst

```diff
@@ -184,7 +185,33 @@ Zigbee
 Common
 ======
 
-There are no entries for this section yet.
+Trusted Firmware-M
+------------------
+
+* Added support for hardware accelerated crypto in TF-M via nrf_security. When enabled (default), calls to psa_crypto APIs will utilize the CryptoCell hardware on nRF9160 and nRF5340.
+* Added support for using hardware unique keys for key derivation (TFM_CRYPTO_ALG_HUK_DERIVATION). TF-M automatically generates and stores random hardware uniques keys if not present, using the hw_unique_key library.
+* TFM_MINIMAL: Added a size-optimized configuration of TF-M which provides a minimal set of features
+  * This configuration requires 32kB of flash and provides random number generation, SHA256 and the platform memory read service.
+  * The configuration is showcased on the TF-M Hello World sample.
+
+Crypto
+------
+
+* nrf_security:
+  * Added functionality to configure and enable crypto hardware acceleration as part of the TF-M build.
+  * Added configurations to enable PSA Crypto APIs in non-TF-M builds.
+  * (EXPERIMENTAL) psa_its: Added a library to provide ITS APIs for using the Zephyr settings subsystem for non-volatile storage of key material. Warning: The storage format is likely to change without backwards compatibility.
+* nrf_cc3xx_platform/nrf_cc3xx_mbedcrypto
+  * Updated the use mbed TLS version 2.26.0 to align with upstream TF-M.
+  * Added low-level APIs for managing and using hardware unique keys located in the KMU peripheral, or flash + K_DR, when no KMU is available.
+  * Added platform APIs for hmac_drbg
+  * Full changelog: `nrfxlib/crypto/nrf_cc3*/CHANGELOG.rst`
+* hw_unique_key: New library for managing and using hardware unique keys, building on the APIs in nrf_cc3xx_platform. HUKs are secret keys that are kept hidden from the application code, but which can be used by the application for deriving keys for other purposes, such as encrypting data for storing.
+
```

Comment:
Let's add the text for the crypto samples under the crypto section, maybe in a "samples" subsection. 

Added samples showcasing the usage of the Platform Security Architecture (PSA) APIs . The samples perform various cryptographic operations such as encryption/decryption using  symmetric and asymmetric ciphers, key exchange,  hashing and random number generation. Both TF-M enabled targets and secure only targets are supported.

Reply:
will do, please don't push here

Reply:
Yep, I will not push here. Feel free to change the wording if you have any suggestions.